eng/pipelines/OpenApiSpecs/GetRepoTree.py

```diff
@@ -0,0 +1,118 @@
+from git import Repo
```

Comment:
move this under `eng/scripts`

Reply:
Moved. I also rename the file to be more descriptive.